packages/core/src/browser/style/sidepanel.css

```diff
@@ -173,3 +180,33 @@
 .p-TabBar.theia-app-right > .theia-TabBar-hidden-content .p-TabBar-tabLabel {
   transform: none;
 }
+
+/*-----------------------------------------------------------------------------
+| Sidepanel Toolbar
+|----------------------------------------------------------------------------*/
+
+.theia-sidepanel-toolbar {
+    min-height: 30px;
+    display: flex;
+    padding-left: 5px;
+    align-items: center;
+}
+
+.theia-sidepanel-toolbar.theia-left-side-panel {
+    border-right: var(--theia-panel-border-width) solid var(--theia-border-color1);
+}
+
+.theia-sidepanel-toolbar.theia-right-side-panel {
+    border-left: var(--theia-panel-border-width) solid var(--theia-border-color1);
+}
+
+.theia-sidepanel-toolbar .theia-sidepanel-title {
+    color: var(--theia-ui-font-color2);
+    flex: 1;
```

Comment:
```css
    margin-left: 14px;
    text-transform: uppercase;
    font-size: 10px;
```

Reply:
@svenefftinge Should not we align title styles in all areas?

Reply:
it would be nice if title styles are aligned when we drop them between panels. i.e. for views they should be in uppercase always and for other widgets (editors, terminals, previews) as in master.